03e_more_ipyautoui_pydantic.ipynb

```diff
@@ -152,46 +168,209 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "ui_data_smooth = AutoUi(DataSelector)\n",
-    "ui_data_smooth"
+    "controls = AutoUi(DataSelector)\n",
+    "controls"
```

Comment:
Can you hint ipyautoui to give enough horizontal space to show the labels for window size and polynomial order without wrapping or cutting off labels?

Reply:
In principle, yes, but in practice not yet.....